components/support/migration/src/main/java/mozilla/components/support/migration/MigrationResultsStore.kt

```diff
@@ -57,6 +57,7 @@ internal class MigrationResultsStore(context: Context) : SharedPreferencesCache<
                 Migration.Bookmarks.javaClass.simpleName -> Migration.Bookmarks
                 Migration.OpenTabs.javaClass.simpleName -> Migration.OpenTabs
                 Migration.Gecko.javaClass.simpleName -> Migration.Gecko
+                Migration.FxA.javaClass.simpleName -> Migration.FxA
```

Comment:
This has happened to me too when I added the code for Gecko. I wish we would have used the power of static types and exhaustive when statements to make the compiler just fail if we miss a case here. Anyhow, for now let"s get that landed. :)